### PR TITLE
Fixed settingsV2  bugs and added experimentation

### PR DESCRIPTION
**Pull request by srinathnarayanan**
This PR does the following
- Replaces settingsv2 feature flag with flights passed on to data explorer initialization 
- Truncates throughput and TTL seconds text input fields when it exceeds the max value
- Fixes a bug due to which auto scale throughput increased by 100 instead of 1000 when the up arrow is pressed
- Adds dirty color change to Autoscale-throughput radio buttons

### CODE REVIEW

File: src/Common/Constants.ts [hunk 1]
```diff
@@ -131,6 +130,10 @@ export class Features {
   public static readonly enableSDKoperations = "enablesdkoperations";
 }
 
+export class Flights {
+  public static readonly settingsV2 = "settingsv2";
```
Comment (swviswan) [inline src/Common/Constants.ts:134]: >settingsv2 [](start = 39, length = 10)

Settingsv2? Can be the flight name with upper case?

File: src/Explorer/Explorer.ts [hunk 2]
```diff
@@ -1953,6 +1956,16 @@ export default class Explorer {
     return Q();
   }
 
+  public setFeatureFlagsFromFlights(flights: readonly string[]): void {
+    if (!flights) {
+      return;
+    }
+
+    if (flights.filter((flightName: string) => flightName === Constants.Flights.settingsV2).length === 1) {
```
Comment (swviswan) [inline src/Explorer/Explorer.ts:1964]: >filter [](start = 16, length = 6)

Can we simply use indexof? filter creates new array which seems unnecessary here.

File: src/Explorer/Explorer.ts [hunk 3]
```diff
@@ -1953,6 +1956,16 @@ export default class Explorer {
     return Q();
   }
 
+  public setFeatureFlagsFromFlights(flights: readonly string[]): void {
+    if (!flights) {
+      return;
+    }
+
+    if (flights.filter((flightName: string) => flightName === Constants.Flights.settingsV2).length === 1) {
+      this.isSettingsV2Enabled(true);
```
Comment (swviswan) [inline src/Explorer/Explorer.ts:1965]: >isSettingsV2Enabled [](start = 11, length = 19)

as discussed yesterday, do you want to turn it on for MPAC - just for this feature so we can find issues faster?

File: src/Common/Constants.ts (hunk 1 above)
Comment (srinathnarayanan) [reply src/Common/Constants.ts:134]: resolved offline, flights returned from experimentation package are lowercase

File: src/Common/Constants.ts [hunk 4]
```diff
@@ -131,6 +130,10 @@ export class Features {
   public static readonly enableSDKoperations = "enablesdkoperations";
 }
 
+export class Flights {
```
Comment (swviswan) [inline src/Common/Constants.ts:133]: >Flights [](start = 13, length = 7)

Can you please add a comment here that all flights from the framework will be lowercase?

**Review by swviswan — APPROVED** [review]
:shipit:

**Review by vignesh-mr — APPROVED** [review]

**Pull request merged**